### DISCUSSION TITLE
Does it works in Outlook e-mail databases?

### DISCUSSION THREAD

**Discussion by rafaelkrapf** [arc53/DocsGPT#34] [Q&A]
Hello! 
1st: Congratulations for the project. It sounds amazing. 
2nd: Can it be trained to search on Outlook e-mail database?

**Comment by dartpain** [comment]
Thank you,

Yes as long as you convert it to txt or rst format and ingest it as shown here https://github.com/arc53/docsgpt/wiki/How-to-train-on-other-documentation
